.gitmodules

```diff
@@ -1,6 +1,6 @@
 [submodule "tests/repositories/rca-output"]
 	path = tests/repositories/rca-output
-	url = https://github.com/mozilla/rust-code-analysis-output
+	url = https://github.com/michaelvanstraten/rust-code-analysis-output
```

Comment:
We have to revert this change

Reply:
https://github.com/mozilla/rust-code-analysis-output/pull/4

Reply:
Merged, thanks! Let's update this PR